public/content/en/gems/uniform-function-call-syntax-ufcs.md

```diff
@@ -16,14 +22,28 @@ It is possible to write
 
     a.bar().foo()
 
+Moreover in D it is not necessary to use parenthesis for functions
+without arguments, which means that _any_ function can be used
+like a property:
+
+    import std.uni : toLower;
+    "D rocks".toLower; // "d rocks"
+
+### Importance of UFCS
+
 UFCS is especially important when dealing with
 *ranges* where several algorithms can be put
 together to perform complex operations, still allowing
 to write clear and manageable code.
 
+    [1, 2].chain([3, 4]).retro; // 4, 3, 2, 1
+    [1, 1, 2, 2, 2].group.dropOne.front; // tuple(2, 3u)
```

Comment:
I think even without explanation, the reader can understand why UFCS is enormously helpful.